internal/controller/snapshot/snapshot_adapter.go

```diff
@@ -34,6 +34,7 @@ import (
 	applicationapiv1alpha1 "github.com/konflux-ci/application-api/api/v1alpha1"
 	"github.com/konflux-ci/integration-service/api/v1beta2"
 	"github.com/konflux-ci/integration-service/gitops"
+	"github.com/konflux-ci/integration-service/helpers"
```

Comment:
Looks like helpers are already imported in this file [here](https://github.com/konflux-ci/integration-service/pull/1054/files#diff-cbf7e6d58cc12a67b910b5b9108ba68d206e75cc8560ca0ab2b19e2d34415702L37).